chart/values.yaml

```diff
@@ -4623,10 +4623,10 @@ externalPostgresql:
   user: workspace_admin
   internalUser: ""
   password: ""
-  adminUser: postgres
```

Comment:
I was thinking and that could be a potential breaking change fo our customers, right?
Maybe we need to leave it as it and open a breaking change ticket to solve it in a bigger release